internal/packages/internal/packageimport/request_manager_test.go

```diff
@@ -114,7 +114,15 @@ func TestRequestManager_DelayedRequests(t *testing.T) {
 		for j := maxRequestIndex; j >= 0; j-- {
 			af := pkg[i].Files
 			bf := pkg[j].Files
-			assert.NotSame(t, af, bf)
+			// Iterate over all file keys in the first file.
```

Comment:
WDYT about somethink like

```suggestion
			// Iterate over all file keys in the first file map and
			// assert that the backing byte slices in af[key] and bf[key] reference differing slices.
```

and dropping the other comments in the loop?